partials/archive-exhibitor/apply.php

```diff
@@ -13,7 +13,7 @@
       if (!empty($exhibitors_apply_heading)) {
     ?>
       <div class="row">
-        <div class="col col-s col-s-12 text-align-center">
+        <div class="col col-s col-s-12 col-m col-m-12 col-l col-l-12 text-align-center">
```

Comment:
because the gutter changes, but when the gutter changes it also resets the flex property. there are some flaws in the boilerplate to be sure